### PR TITLE
Remove debug output when coverage fails

### PR DESCRIPTION
**Pull request by silverwind**
When `make coverage` fails, it logs megabytes of debug to stdout, which seems to break the drone ui as well as the log output download in drone, presumably because drone limits downloadable log size to 5MB. I think with removal of this print, we should still see any errors that gocovmerge.go logs to stderr, but a few CI runs may be necessary to get it to fail again.

Example failures:

https://drone.gitea.io/go-gitea/gitea/58933/2/18
https://drone.gitea.io/go-gitea/gitea/58928/2/17

### CODE REVIEW

**Comment by silverwind** [comment]
Debug output was added in https://github.com/go-gitea/gitea/commit/c0f5da3e1adfe40cafc57de63948ef1878bd75c9 by @zeripath, but I think as it stands now, it's not useful. It may become useful if we can increase this 5MB limit of drone, thought.

File: Makefile [hunk 1]
```diff
@@ -364,7 +364,7 @@ test\#%:
 coverage:
 	grep '^\(mode: .*\)\|\(.*:[0-9]\+\.[0-9]\+,[0-9]\+\.[0-9]\+ [0-9]\+ [0-9]\+\)$$' coverage.out > coverage-bodged.out
 	grep '^\(mode: .*\)\|\(.*:[0-9]\+\.[0-9]\+,[0-9]\+\.[0-9]\+ [0-9]\+ [0-9]\+\)$$' integration.coverage.out > integration.coverage-bodged.out
-	$(GO) run build/gocovmerge.go integration.coverage-bodged.out coverage-bodged.out > coverage.all || (echo "gocovmerge failed"; echo "integration.coverage.out"; cat integration.coverage.out; echo "coverage.out"; cat coverage.out; exit 1)
+	$(GO) run build/gocovmerge.go integration.coverage-bodged.out coverage-bodged.out > coverage.all)
```
Comment (silverwind) [inline Makefile]:
```suggestion
	$(GO) run build/gocovmerge.go integration.coverage-bodged.out coverage-bodged.out > coverage.all
```

**Comment by zeripath** [comment]
If you remove this logging then we have no way of working out why the coverage is failing. 

Do we understand why coverage keeps failing at present?

**Comment by silverwind** [comment]
> Do we understand why coverage keeps failing at present?

No, but maybe a few re-runs of this will reveal the error. As is stands, the debug output is useless as long as drone truncates it to 5MB.

By the way, what is the point of this coverage task? Is it needed for external tools like CodeCov?

**Comment by silverwind** [comment]
I see that these tasks actually uploads to codecov. Still I wonder the value. I think most contributors ignore code coverage and these codecov comments are generally perceived as spam because the percentages are not stable.

I think we should just drop this codecov integration as it introduces unnecessary build failures. Anything that can fail a build randomly should be eliminated 😉.

**Review by Gusted — APPROVED** [review]
Nobody is interested or actively working with coverage, don't see any justification to have this code.

**Review by delvh — APPROVED** [review]

**Pull request merged**